build/upload_artifacts_gh.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+#
+# Upload dist packages to GH Release assets
+#
+#
+cd dist
+for package in $(find  -regex ".*\.\(msi\|rpm\|deb\|zip\|tar.gz\)");do
+  echo "===> Uploading to GH $TAG: ${package}"
```

Comment:
why do we need this, doesn't goreleaser already add the assets to the github release?

Reply:
Goreleaser doesn't build the packages exactly like we currently have. I believe the zip/tar is different so we need to alter it before uploading to GH releases :(

Reply:
correct